### PR TITLE
Changed eservices url composition logic

### PR DESCRIPTION
**Pull request by gdellorusso**
Dev environment test

### CODE REVIEW

File: src/main/java/it/pagopa/interop/probing/caller/util/ClientUtil.java [hunk 1]
```diff
@@ -62,8 +66,9 @@ private TelemetryDto callRest(TelemetryDto telemetryResult, EserviceContentDto s
     long before = System.currentTimeMillis();
     telemetryResult.checkTime(String.valueOf(before));
     AWSXRay.beginSubsegment("Rest_call_to :".concat(service.basePath()[0]));
-    Response response = restClientConfig.feignRestClient()
-        .probing(URI.create(service.basePath()[0]), jwtBuilder.buildJWT(service.audience()));
+    Response response = restClientConfig.feignRestClient().probing(
+        URI.create(service.basePath()[0] + ProjectConstants.PROBING_ENDPOINT_SUFFIX),
```
Comment (galales) [inline src/main/java/it/pagopa/interop/probing/caller/util/ClientUtil.java]: may you remove the potential trailing `/` in the base path?

File: src/main/java/it/pagopa/interop/probing/caller/util/ClientUtil.java [hunk 2]
```diff
@@ -75,9 +80,9 @@ private TelemetryDto callSoap(TelemetryDto telemetryResult, EserviceContentDto s
     long before = System.currentTimeMillis();
     telemetryResult.checkTime(String.valueOf(before));
     AWSXRay.beginSubsegment("Soap_call_to :".concat(service.basePath()[0]));
-    ProbingResponse response =
-        soapClientConfig.feignSoapClient().probing(URI.create(service.basePath()[0]),
-            o.createProbingRequest(), jwtBuilder.buildJWT(service.audience()));
+    ProbingResponse response = soapClientConfig.feignSoapClient().probing(
+        URI.create(service.basePath()[0] + ProjectConstants.PROBING_ENDPOINT_SUFFIX),
```
Comment (galales) [inline src/main/java/it/pagopa/interop/probing/caller/util/ClientUtil.java]: same

Comment (gdellorusso) [reply src/main/java/it/pagopa/interop/probing/caller/util/ClientUtil.java]: done

File: src/main/java/it/pagopa/interop/probing/caller/util/ClientUtil.java (hunk 1 above)
Comment (gdellorusso) [reply src/main/java/it/pagopa/interop/probing/caller/util/ClientUtil.java]: done

**Review by galales — APPROVED** [review]

**Review by nttdata-rtorsoli — APPROVED** [review]

**Pull request merged**